core/domain/improvements_domain_test.py

```diff
@@ -50,35 +42,39 @@ def setUp(self) -> None:
 
     def test_task_id_has_expected_value(self) -> None:
         task_entry = improvements_domain.TaskEntry(
-            improvements_models.TASK_ENTITY_TYPE_EXPLORATION, self.exp_id, 1,
-            improvements_models.TASK_TYPE_HIGH_BOUNCE_RATE,
-            improvements_models.TASK_TARGET_TYPE_STATE,
+            constants.TASK_ENTITY_TYPE_EXPLORATION, self.exp_id, 1,
+            constants.TASK_TYPE_HIGH_BOUNCE_RATE,
+            constants.TASK_TARGET_TYPE_STATE,
             feconf.DEFAULT_INIT_STATE_NAME, 'issue description',
-            improvements_models.TASK_STATUS_RESOLVED, self.owner_id,
+            constants.TASK_STATUS_RESOLVED, self.owner_id,
             self.MOCK_DATE)
         self.assertEqual(
             task_entry.task_id,
             'exploration.eid.1.high_bounce_rate.state.Introduction')
 
     def test_composite_entity_id_has_expected_value(self) -> None:
         task_entry = improvements_domain.TaskEntry(
-            improvements_models.TASK_ENTITY_TYPE_EXPLORATION, self.exp_id, 1,
-            improvements_models.TASK_TYPE_HIGH_BOUNCE_RATE,
-            improvements_models.TASK_TARGET_TYPE_STATE,
+            constants.TASK_ENTITY_TYPE_EXPLORATION, self.exp_id, 1,
+            constants.TASK_TYPE_HIGH_BOUNCE_RATE,
+            constants.TASK_TARGET_TYPE_STATE,
             feconf.DEFAULT_INIT_STATE_NAME, 'issue description',
-            improvements_models.TASK_STATUS_RESOLVED, self.owner_id,
+            constants.TASK_STATUS_RESOLVED, self.owner_id,
             self.MOCK_DATE)
         self.assertEqual(task_entry.composite_entity_id, 'exploration.eid.1')
 
     def test_to_dict_has_expected_value(self) -> None:
+        # Data url for images/avatar/user_blue_72px.png.
+        # Generated using utils.convert_png_to_data_url.
         task_entry = improvements_domain.TaskEntry(
-            improvements_models.TASK_ENTITY_TYPE_EXPLORATION, self.exp_id, 1,
-            improvements_models.TASK_TYPE_HIGH_BOUNCE_RATE,
-            improvements_models.TASK_TARGET_TYPE_STATE,
+            constants.TASK_ENTITY_TYPE_EXPLORATION, self.exp_id, 1,
+            constants.TASK_TYPE_HIGH_BOUNCE_RATE,
+            constants.TASK_TARGET_TYPE_STATE,
             feconf.DEFAULT_INIT_STATE_NAME, 'issue description',
-            improvements_models.TASK_STATUS_RESOLVED, self.owner_id,
+            constants.TASK_STATUS_RESOLVED, self.owner_id,
             self.MOCK_DATE)
-        self.assertEqual(task_entry.to_dict(), {
+        self.assertEqual(
+            task_entry.to_dict(),
+            {
```

Comment:
Either do 
```suggestion
        self.assertEqual(task_entry.to_dict(), {
```

Reply:
move the lines below by 4 spaces to the right

Reply:
Done. The `resolver_id` won't be `None` in non-test files, it will always be the `user_id`.

Reply:
Wait, looking at the `TaskEntryModel` it seems that it can be `None` even outside tests.

Reply:
Yes, you are right, it is None in some functions in `TaskEntryModel`.